gui/system/views.py

```diff
@@ -998,7 +998,8 @@ def debug(request):
                 with client as c:
                     c.call('failover.call_remote', 'core.ping')
             except ClientException:
-                return render(request, 'failover/failover_down.html')
+                return render(request, 'system/debug.html', {"failover_availability" : False})
```

Comment:
Exactly - I've just written it like that but I made some mistake during git operations (it is previous version). Sorry for that